lib/config.js

```diff
@@ -28,10 +28,16 @@ async function getGitRev () {
 async function getAppiumConfig () {
   let stat = await fs.stat(path.resolve(__dirname, '..'));
   let built = stat.mtime.getTime();
+
+  // this code can be run pre-transpiled (so, code is in 'build')
+  // or transpiled on the fly (so, code is in root directory)
+  let {version} = __filename.indexOf(path.join('build', 'lib', 'config')) !== -1 ?
```

Comment:
probably, it would be better to extract it into some common method to avoid unnecessary duplication?